qt_ui/widgets/QIntelBox.py

```diff
@@ -24,30 +24,49 @@ def __init__(self, game: Game) -> None:
         self.setLayout(columns)
 
         summary = QGridLayout()
-        columns.addLayout(summary)
+        summary.setContentsMargins(5, 5, 5, 5)
+        # columns.addLayout(summary)
 
-        summary.addWidget(QLabel("Air superiority:"), 0, 0)
+        air_superiority = QLabel("Air superiority:")
+        summary.addWidget(air_superiority, 0, 0)
         self.air_strength = QLabel()
         summary.addWidget(self.air_strength, 0, 1)
 
-        summary.addWidget(QLabel("Front line:"), 1, 0)
+        front_line = QLabel("Front line:")
+        summary.addWidget(front_line, 1, 0)
         self.ground_strength = QLabel()
         summary.addWidget(self.ground_strength, 1, 1)
 
-        summary.addWidget(QLabel("Economic strength:"), 2, 0)
+        economy = QLabel("Economic strength:")
+        summary.addWidget(economy, 2, 0)
         self.economic_strength = QLabel()
         summary.addWidget(self.economic_strength, 2, 1)
 
-        details = QPushButton("Details")
-        columns.addWidget(details)
-        details.clicked.connect(self.open_details_window)
+        # some dirty styling to make the labels show up well on the button
+        button_text_style = "background-color: rgba(0,0,0,0%); color: white;"
+        air_superiority.setStyleSheet(button_text_style)
+        front_line.setStyleSheet(button_text_style)
+        economy.setStyleSheet(button_text_style)
+        self.air_strength.setStyleSheet(button_text_style)
+        self.ground_strength.setStyleSheet(button_text_style)
+        self.economic_strength.setStyleSheet(button_text_style)
+
+        # details = QPushButton("Details")
+        self.details = QPushButton()
```

Comment:
Is this doing what I think it's doing? The button is big and empty, and the labels are just placed on top of it?

Reply:
I moved the layout for the labels to be a child in the button, but it appears the QPushButton doesn't take size hints from its children. Without using the setMinimumHeight/Width methods the button doesn't size properly and hides the text inside it.

So the labels are inside the button and therefore act as if they were the normal label on it rather than blocking input when on top.

Reply:
Ah, I see `summary` is the layout of the button. Cool, I didn't know you could do that.